covid_graphs/covid_graphs/country_report.py

```diff
@@ -4,24 +4,17 @@
 from pathlib import Path
 from typing import List
 
-from .formula import Formula
 from .pb.country_data_pb2 import CountryData
 
 
-@dataclass
-class Country:
-    name: str
-    formulas: List[Formula]
-
-
 class CountryReport:
-    def __init__(self, data_dir: Path, country: Country):
+    def __init__(self, data_dir: Path, country_name: str):
         """Constructs a numpy representation of data read from 'data_dir' a given country."""
         country_data = CountryData()
-        data_file_path = Path(data_dir / f'{country.name}.data')
+        data_file_path = Path(data_dir / f'{country_name}.data')
         text_format.Parse(data_file_path.read_text(), country_data)
 
-        self.name = country.name
+        self.name = country_name
```

Comment:
Just a note here: these are not the same names.

- country_name is used for data storage, so we have `UK.data` or 'USA.data`.
- country.name is longer, we store `United Kingdom` there, for example.

So I'd prefer keeping it the old way, and renaming `country_name` to `short_country_name`.

Reply:
Nice catch!

`Country` should probably became a class as well (storing names/ids for various purposes). Leaving for later PRs.